posthog/hogql/grammar/HogQLParser.g4

```diff
@@ -117,6 +118,7 @@ joinConstraintClause
     ;
 
 sampleClause: SAMPLE ratioExpr (OFFSET ratioExpr)?;
+limitExpr: columnExpr ((COMMA | OFFSET) columnExpr)?;
```

Comment:
Since this is specific to `limitByClause` – only used there and not general to LIMIT like the `limitExpr` name suggests – feels more readable to inline there

Reply:
I banged my head against the wall for a while, but easier to do like this